<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="minimumSize">
        <size>
         <width>973</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Current Data</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>256</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <widget class="QLabel" name="label_SulfDioxide">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dioxido de azufre (SO2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_CarbonMonoxide">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Monox de carbono (CO)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_lel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Indice de explosividad (%LEL)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_temperature">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Temperatura</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>191</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLCDNumber" name="lcdNumber_sulfDioxide"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_carbonMonoxide"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_lel"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_temperature"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_Picture">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>200</y>
          <width>551</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_Picture</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>layoutWidget</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Historical Data</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="horizontalFrame_chart">
          <layout class="QHBoxLayout" name="horizontalLayout_chart"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_serialStart">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_serialBaudRate">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_serialPorts">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>210</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_serialBaudRate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Boud Rate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_serialPort">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>210</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Port (COM)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_serialClose">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>210</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButto_deleteAllDB">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>170</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Clean Data Base</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
    <addaction name="actionSign_up"/>
   </widget>
   <addaction name="menuLogin"/>
  </widget>
  <action name="actionSign_up">
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
